horace_core/sqw/file_io/@faccess_sqw_v4/get_sqw.m

```diff
@@ -114,7 +128,7 @@
     else
         sqw_object   = sqw_skel.main_header;
         varargout{1} = sqw_skel.experiment_info;
-        varargout{2} = sqw_skel.detpar;
+        varargout{2} = [];
```

Comment:
well, if your `sqw.detpar` accepts proper assignment (`detector_array`) this should remain. Why do you need to remove it here?

Reply:
I've put an explanation for this in the file. sqw_skel is not a true sqw but a struct with the same data in it. as sqw no longer has a detpar value (it is used to transfer detector_array info) there should be nothing here. None of our tests reach this line so I am unable to determine what the output should look like. hopefully one day this will throw an error and we can catch it